templates/engage/14-04-esm.html

```diff
@@ -23,7 +34,16 @@ <h1>Still running Ubuntu 14.04?</h1>
       </p>
     </div>
     <div class="col-4 u-hide--small prefix-1">
```

Comment:
can you change this too? prefix-1 is gone a
```suggestion
    <div class="col-4 u-hide--small u-vertically-center u-align--center">
```